src/java.base/share/classes/module-info.java

```diff
@@ -214,17 +214,21 @@
         java.net.http,
         java.rmi,
         java.security.jgss,
+        java.smartcardio,
         jdk.attach,
         jdk.charsets,
         jdk.compiler,
         jdk.crypto.cryptoki,
         jdk.incubator.concurrent,
         jdk.incubator.vector,
         jdk.jfr,
+        jdk.jpackage,
         jdk.jshell,
+        jdk.net,
         jdk.nio.mapmode,
         jdk.unsupported,
-        jdk.internal.vm.ci;
+        jdk.internal.vm.ci,
+        jdk.zipfs;
```

Comment:
We shouldn't be exporting jdk.internal.misc to all these modules, need to find a better place for this class.